extensions/amp-social-share/1.0/social-share.js

```diff
@@ -24,33 +24,36 @@ import {getSocialConfig} from './social-share-config';
 import {openWindowDialog} from '../../../src/dom';
 import {useResourcesNotify} from '../../../src/preact/utils';
 
-const DEFAULT_WIDTH = 60;
-const DEFAULT_HEIGHT = 44;
-const DEFAULT_TARGET = '_blank';
 const NAME = 'SocialShare';
+const DEFAULT_WIDTH = '60';
```

Comment:
Why do these need to become strings?

Reply:
Initially I had wanted to set the `width` and `height` props to `(string|undefined)`, so wanted to follow that format throughout the code.

In the latest, I decided to change it to `(number|string|undefined)` so this is reflected in the latest revision.  Thanks!!